data-source/jsonnet/england-wales/household/blocks/who-lives-here/any_visitors.jsonnet

```diff
@@ -13,6 +13,7 @@ local questionTitle(census_date) = {
 function(census_date) {
   type: 'Question',
   id: 'any-visitors',
+  show_on_section_summary: false,
```

Comment:
I know this was discussed in the office, but didn't mention it as I felt it should be part of another PR. Has anyone actually requested we should flip this flag or what it should be named?

Reply:
I spoke to @ajmaddaford about it and agreed to flip it